app/src/main/res/values/strings.xml

```diff
@@ -166,6 +166,8 @@
     <string name="cancel_button">Cancel</string>
     <!-- This string labels a button that are used to approve an action. -->
     <string name="ok_button">OK</string>
+    <!-- This string labels a button that are used to approve an action. -->
```

Comment:
Maybe "This string labels a button to go back to the previous menu or exit the current mode"?